snippets/base/templates/base/includes/snippet_js.html

```diff
@@ -319,6 +320,21 @@
         }
     }
 
+    // Add links to buttons that block snippets.
+    function addSnippetBlockLinks(elements) {
+        var snippet_id = document.getElementById('snippet_set').getAttribute('data-snippet-id');
+        var blockSnippet = function (event) {
+            event.preventDefault();
+            addToBlockList(snippet_id);
+            sendMetric('snippet-blocked', function() { window.location.reload(); })
```

Comment:
Someday we might want to do the reload first to avoid making the user wait for the metric request, but this is fine too.